<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updBtn</class>
 <widget class="QMainWindow" name="updBtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="seller_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All avaliable sellers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellerDocx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all sellers *.docx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellerExcel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all sellers *.excel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search seller by surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entersur">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>surname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serchSellerBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListView" name="searchSeller">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>100</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new seller</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="midName">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter middle name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percent">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter income percent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSeller">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manipulate_sellerBtn">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Manipulate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manipulate_surname">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>surname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updSellers">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update list</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
